common/pigx-runner.in

```diff
@@ -214,14 +214,17 @@ settings.
         defaults = path.join(dirs['locations']['pkgdatadir'], 'settings.yaml')
 
     if not path.exists(defaults):
-        bail("Could not find default settings.  Did you forget to set PIGX_UNINSTALLED?")
+        bail(
+            f"Could not find default settings at {defaults}. Did you forget "
+            f"to set PIGX_UNINSTALLED?"
+        )
```

Comment:
As a Lisper these lonely parentheses break my heart.   But I shall not mention it again :)

Reply:
Well I am still finding my way... The default config for R `styler` seems to be this style and so I am currently using it for consistency, but I am not married to it. Is the preferred way for you to never have lonly parens in their own line, always at the end of text?

Reply:
Is it actually a good idea to use the R styler for python code?
Maybe we should agree on a specific linter for each language that we are using, i.e. Black for python, styler for R?

Reply:
Black may be a little bit annoying for this because it doesn't do partial formatting, and will choke on these autotools infiles. But even so, it doesn't seem to mind the formatting when I apply it to the finished `pigx-sars-cov-2` script. I think we need a broader discussion for what linters to use with which settings before settling on one. But yeah of course it doen't make too much sense to apply the `R` linter to the `python` file...